man/coolconfig.1

```diff
@@ -8,7 +8,7 @@ coolconfig COMMAND [OPTIONS]
 .SS "General options:"
 \fB\-h\fR, \fB\-\-help\fR                Show this usage information.
 .PP
-\fB\-\-config\-file\fR=\fIpath\fR        Specify configuration file path manually.
+\fB\-\-config\-file\fR=\fIpath\fR        Specify configuration file path manually. The default is \fI/etc/coolwsd/coolwsd.xml\fR.
```

Comment:
Not always. See https://github.com/CollaboraOnline/online/blob/0340dbd8938a0ab36f369d93cc8696842361c08c/tools/Config.cpp#L89 On FreeBSD I think it's not /etc.

Reply:
How about: `The default is the path to coolwsd.xml`?

Reply:
The default is usually _/etc/coolwsd/coolwsd.xml_. We covered 99.9% of the cases.